File: Frontend/library-app/src/models/SuccessPurchaseRequest.ts
class SuccessPurchaseRequest {
  paymentKey: string;
  orderId: string;
  purchaseDate: string;
  status: string;
  totalAmount: number;
  constructor(
    paymentKey: string,
    orderId: string,
    purchaseDate: string,
    status: string,
    totalAmount: number
  ) {
    this.paymentKey = paymentKey;
    this.orderId = orderId;
    this.purchaseDate = purchaseDate;
    this.status = status;
    this.totalAmount = totalAmount;
  }
}
export default SuccessPurchaseRequest;

File: Backend/spring-boot-library/src/main/java/com/reactlibraryproject/springbootlibrary/RequestModels/SuccessPurchaseRequest.java
package com.reactlibraryproject.springbootlibrary.RequestModels;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class SuccessPurchaseRequest {
    private String paymentKey;
    private String orderId;
    private String purchaseDate;
    private String status;
    private int totalAmount;
}
